app/controllers/api/v1/cars_controller.rb

```diff
@@ -21,6 +21,27 @@ def create
     end
   end
 
+  # PUT /cars/:id
+  def update
+    if @car
+      @car.update(car_params)
+      render json: { message: 'Car successfully updated.' }, status: :ok
+    else
+      render json: { error: 'Unable to update car.' }, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /cars/:id
+  def destroy
+    @car = Car.find(params[:id])
+    if @car
+      @car.destroy
```

Comment:
Please, can you confirm what happens if the destroy method fails? According to the code, the client will receive a success message and only receives an error message if the car doesn't exist.

Reply:
I fixed. Now when you tried to delete a car that doesn't exits the api answer with a message:

![image](https://user-images.githubusercontent.com/10566417/236587021-06ac0758-cf85-4861-85e2-7e8c713fbd7f.png)